go get -v doesn't work against master

##### ISSUE TYPE
 - [X ] Bug Report

# BUG REPORT

##### SUMMARY
Following the source installation process fails

##### ENVIRONMENT
* Kubeaudit version: master
* Kubeaudit install method: DIY-BUILD

##### STEPS TO REPRODUCE

Run `go get -v github.com/Shopify/kubeaudit`

##### EXPECTED RESULTS

the application downloads/compiles.

##### ACTUAL RESULTS

`github.com/Shopify/kubeaudit/scheme/register.go:54:21: cannot use "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1".AddToScheme (type func(*"github.com/jetstack/cert-manager/vendor/k8s.io/apimachinery/pkg/runtime".Scheme) error) as type func(*"k8s.io/apimachinery/pkg/runtime".Scheme) error in array or slice literal`

##### ADDITIONAL INFORMATION

`go version go1.10.4 linux/amd64`

--- a/README.md
+++ b/README.md
@@ -23,10 +23,6 @@ privileged, ... You get the gist of it and more on that later. Just know:
 
 ## Installation
 
-> **Please note `go get kubeaudit` is currently not functional**
-> 
-> We are aware of the issue and are working on a fix. You can still get kubeaudit using these alternate methods:
-
 #### Download a binary
 
 Kubeaudit has official releases that are blessed and stable here:
@@ -38,8 +34,18 @@ Master will have newer features than the stable releases. If you need a newer
 feature not yet included in a release you can do the following to get
 kubeaudit:
 
+For go 1.12 and higher:
+```sh
+GO111MODULE=on go get -v github.com/Shopify/kubeaudit
+cd $GOPATH/src/github.com/Shopify/kubeaudit
+make
+make install
+```
+
+For older versions of go:
 ```sh
 git clone https://github.com/Shopify/kubeaudit.git
+cd kubeaudit
 make
 make install
 ```

--- a/README.md
+++ b/README.md
@@ -37,9 +37,6 @@ kubeaudit:
 For go 1.12 and higher:
 ```sh
 GO111MODULE=on go get -v github.com/Shopify/kubeaudit
-cd $GOPATH/src/github.com/Shopify/kubeaudit
-make
-make install
 ```
 
 For older versions of go:

--- a/go.mod
+++ b/go.mod
@@ -26,3 +26,5 @@ require (
 	k8s.io/client-go v0.0.0-20190508063711-1babf78c8b32
 	k8s.io/utils v0.0.0-20190308190857-21c4ce38f2a7 // indirect
 )
+
+go 1.13

